README.md

```diff
@@ -108,3 +108,10 @@ think the issue should get addressed.
 - **Priority**. Each issue has a priority which is represented by the column in the [Prioritization](https://github.com/orgs/istio/projects/6) project. Priority can be one of
 P0, P1, P2, or >P2. The priority indicates how important it is to address the issue within the milestone. P0 says that the
 milestone cannot be considered achieved if the issue isn't resolved.
+ 
```

Comment:
```suggestion

```

Reply:
Were you trying to remove the blank line or spaces in it?  (I think spaces in a blank line are meant to have meaning in Markdown, but I don't think GHFM cares.)